kitty-items-cadence/contracts/Kibble.cdc

```diff
@@ -1,4 +1,4 @@
-import FungibleToken from "./FungibleToken.cdc"
+import FungibleToken from 0xee82856bf20e2aa6
```

Comment:
Emulator FT address.

Reply:
Check out this feature: https://docs.onflow.org/flow-cli/initialize-project/#advanced-format

You can actually specify contract aliases in the `flow.json` file